source/auth_n.rst

```diff
@@ -5101,15 +5049,6 @@ If authentication is successful, you will get back the Account:
       .. literalinclude:: code/vbnet/authentication/mfa_auth_account_resp.vb
           :language: vbnet
 
-.. only:: java
-
-  .. todo::
-
-    (java.todo)
-
-    .. literalinclude:: code/java/authentication/mfa_auth_account_resp.java
-        :language: java
-
```

Comment:
@jakubswiatczak In java is it a bit different, the "request/response" scheme as applies to REST does not quite apply in java. In java things are more implicit
For example in `mfa_add_sms_factor_req.java` you get a `Factor` back by calling `createFactor()` method which is a bit different from REST point of view:
`factor = account.createFactor(factor);` 
Therefor files like `mfa_auth_account_resp.java` wouldn't make much sense in this context.

Thanks,
Mehrshad